contrib/automation_tests/fragments/bottom_up_tab.py

```diff
@@ -0,0 +1,48 @@
+"""
+Copyright (c) 2020 The Orbit Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+"""
+import logging
+
+from core.orbit_e2e import E2ETestCase, find_control
+
+
+class VerifyHelloGgpBottomUpContents(E2ETestCase):
+    """
+    TODO: This is a more or less plain copy of the old E2E test. If may need to
+    be updated
+    """
```

Comment:
Same here: how does it need to be updated?

Reply:
Same as above